### ISSUE TITLE
Unhandled Exception: MissingPluginException(No implementation found) on Android

### ISSUE THREAD

**Issue by adityadroid** [Instabug/Instabug-Flutter#123]
Anyone else facing the issue on Android release builds, disabling minification in `build.gradle` solved the issue for us. 

```
        release {
			minifyEnabled false
         }
```

This relates to #78 #108 #118 and #110 .

**Comment by knyghtryda** [comment]
This still took quite a bit of work as it required also adding 
```gradle
shrinkResources false
```
to `release` and downgrading gradle back to 3.5.3.  Finally, you need to make sure that `android.enableJetifier=true` is NOT in your gradle.properties.  Instabug, any ideas as to whether this will be fixed or addressed?

**Comment by subhash279** [comment]
We were having the same issue. Instabug.show() wasn't working in Android release builds but was working fine in debug builds. It was resolved only after adding
`minifyEnabled false
  shrinkResources false`
in buildTypes -> release {...

**Comment by AliAbdelfattah** [comment]
Thank you guys for confirming this. We're looking into it.

**Comment by knyghtryda** [comment]
FYI, this is really important to my current project, as disabling jetifier is causing Google sign-in to fail.  I can't keep using Instabug if its causing my Google Sign-in to hard crash.

**Comment by subhash279** [comment]
@AliAbdelfattah just checking if there is any progress on this. Disabling minify increases app size by 20-25%. Please let us know if there is a rough estimate on when this will be fixed.

**Comment by AliAbdelfattah** [comment]
@subhash279 a fix has been applied and should be included in the upcoming release. Expected within a couple of days.

**Comment by subhash279** [comment]
@AliAbdelfattah  thanks for letting us know. Just checking is this already included in the latest release on pub dev.

**Comment by AliAbdelfattah** [comment]
@subhash279 fix has been released now in v9.1.6. Closing the issue, feel free to reopen if you face it again.
